content/millennium/r4/financial/charge-item.md

```diff
@@ -69,6 +69,43 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `revenue-code`         | None (contains nested extensions) | The type of revenue or cost center providing the product and/or service.
  `unit-price`           | [`Money`]                         | The price of a single unit for the resource.
 
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for ChargeItems that meet supplied query parameters:
+
+    GET /ChargeItem?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name                  | Required?  | Type          | Description
+-----------------------|------------|---------------|-------------------------------------------------------
+ `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+
```

Comment:
Should we add a Notes section here stating something about only supporting a single _id at a time?

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/653/commits/2263410ecae10e46708116d1a4790f2582133e1b#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR90-R93

Reply:
+1